### DISCUSSION TITLE
Align with Community and keep 2023.5.8 as Insomnia and version 8.x should be an enterprise version

### DISCUSSION THREAD

**Discussion by meshpaul** [Kong/insomnia#6860] [Ideas] [answered]
The current product has changed dramatically.  While this may be great for enterprises and some features that are added to build revenue is understandable.  There is big portion of folks whant the Non mandatory account creation version that was offered in the Version: Insomnia 2023.5.8 Build date: 9/7/2023.  IMHO 8.0 is not suitable for Small teams/Start-ups and  needs an enterprise version of it that added as part of 8.X features.  While 8.3 and beyond tries to address these concerns the trust is lost in the product.  Recommend to roll out ether less invasive product that is similar to 2023.5.8 Build date: 9/7/2023

**Comment by subnetmarco** [comment]
Hi @meshpaul newer versions of Insomnia are not only great for enterprises, the latest versions of Insomnia have had the highest usage of any other Insomnia version ever. We are really pouring a ton of resources here.

I really want to understand something that is not clear to me, and I would like to be as non-provocative or offensive as possible by asking the following question, because it truly is for my personal understanding. 

Insomnia is a product that delivers a certain amount of value to its users, arguably your are one of them. But building Insomnia is not free, we depend on the conversion of a certain amount of users to a paid plan. As such, even in its free version, we ask for an account in the hope that our premium capabilities will allow to convert a subset of our user-base into a paid plan which will further drive development of both the open source and premium versions of Insomnia.

Insomnia without a customer base - like every other product in the world - just cannot exist as a product in any capacity, including OSS.

As such, just so I understand better, what is it that is so repulsing about creating a free account on Insomnia?

**Reply by snuggeman** [reply]
Privacy? Annoyance? Workspace restrictions?

And every product does NOT have to have customers. But USERS are very nice to have.

**Reply by domigi** [reply]
I would be willing to pay around 5$ to purchase a one time lifetime license per device that allows me to use insomnia without an account and work independent from your cloud availability.

**Comment by emschu** [comment]
I need a tool, not a product

**Comment by matthewcaminiti** [comment]
I wholeheartedly agree with @meshpaul and @emschu .

I've been using Insomnia for years and have been an ardent promoter to many friends and colleagues. The 8.0 version has turned a beautifully simple tool into a worse-than-competitors product that I will now recommend others to avoid.

A return to the no-strings-attached 2023 build would be best

**Comment by subnetmarco** [comment, marked as answer]
Since we decided to make this change, we were able to finally find a more sustainable path for Insomnia, as demonstrated by the numerous releases and features introduced since this change was made. 

At the same time we are still offering most capabilities for free, Insomnia's entire offering is now SOC2 certified, with optional support for E2EE for all user data, and the ability to store data **100% locally**, in the cloud or on Git based on the storage settings of your projects. Additionally all account data is secured by a system compliant with ISO27001, SOC2, ISO27018, HIPAA BAA, Gold CSA STAR and PCI DSS compliances.

We also continue to offer Scratch Pad **without having to create an account** for individual users that simply need a visual API client. 

To support this continued product development and community support, the changes we introduced a year ago when it comes to requiring user accounts in exchange for the fuller product experience, allowed us to find the right balance while still continuing to offer quite a lot of value in the free edition of Insomnia. We think that you - the free user - are still prioritized in both the breadth and depth of features that we continue to deliver.

**Comment by magnus-freshx** [comment]
For those of you thinking: ooh, finally a version that does not require an account where I can use my collections from 2023.5.8:

Don't bother, you still can't use them locally. Scratch pad limited to 1 collection.
